medusa/server/web/core/calendar.py

```diff
@@ -26,6 +26,7 @@ def get(self):
         """
         Render the iCalendar
         """
+        self.add_header("Content-Type", "text/calendar")
```

Comment:
please change quotes to single. We reserve double for docstrings. Or to encapsulate single quotes in strings.

Reply:
Done